git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -42,13 +42,15 @@ script: https://kit.fontawesome.com/83b2343bd4.js
 
 **Estimated time to completion:**
 
+@estimated_time
+
 **Pre-requisites**
 
 Before you start this module, make sure you
 
 * Know how to access a command line interface (CLI) on your computer.
-* Configure Git on your computer.
-* The ability to edit plain text `.txt` documents. A text editor is different from a word processor (i.e. Microsoft Word or Google Docs), in that text editors create files that consist only of text, no formatting at all. Here is a [tutorial on editing text files using `nano`](https://swcarpentry.github.io/shell-novice/03-create/#create-a-text-file), one of many text editors that you can access directly from the command line interface (CLI).
+* Have Git configured on your computer.
```

Comment:
Just a note here to link this to git install and setup modules when those come on board, no action item necessary rn

Reply:
They are now up, done!